src/main/java/gmail/taras/shevchuk0024/lesson37_Collections/NumberUtils.java

```diff
@@ -0,0 +1,26 @@
+package gmail.taras.shevchuk0024.lesson37_Collections;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NumberUtils {
+
+    public static int getSum(List<Integer> numbers) {
+        int sum = 0;
+        for (int number : numbers) {
+            sum += number;
+        }
+        return sum;
+    }
+
+    public static List<Integer> multiplyOddNumber(List<Integer> numbers) {
+        List<Integer> result = new ArrayList<>();
+        for (int number : numbers) {
+            if (number % 2 != 0) {
+                result.add(number * 2);
+            }
+        }
+        return result;
+    }
```

Comment:
Краще використати long замість int :) В такому випадку Ваш метод зможе корректно опрацювати масив з інтів та лонгів. Плюс в завданні вказано що треба створити метод multiplyOddNumber(List<**Number**> numbers).

```suggestion
    public static List<Long> multiplyOddNumber(List<Number> numbers) {
        List<Long> result = new ArrayList<>();
        for (Number number : numbers) {
            if (number.longValue() % 2 != 0) {
                result.add(number.longValue() * 2);
            }
        }
        return result;
    }
```